invokeai/app/services/model_load/model_load_default.py

```diff
@@ -75,3 +83,41 @@ def load_model(self, model_config: AnyModelConfig, submodel_type: Optional[SubMo
             self._invoker.services.events.emit_model_load_complete(model_config, submodel_type)
 
         return loaded_model
+
+    def load_model_from_path(
+        self, model_path: Path, loader: Optional[Callable[[Path], Dict[str, Tensor]]] = None
+    ) -> LoadedModel:
+        cache_key = str(model_path)
+        ram_cache = self.ram_cache
+        try:
+            return LoadedModel(_locker=ram_cache.get(key=cache_key))
+        except IndexError:
+            pass
+
+        def torch_load_file(checkpoint: Path) -> Dict[str, Tensor]:
+            scan_result = scan_file_path(checkpoint)
+            if scan_result.infected_files != 0:
+                raise Exception("The model at {checkpoint} is potentially infected by malware. Aborting load.")
+            result: Dict[str, Tensor] = torch_load(checkpoint, map_location="cpu")
+            return result
+
+        def diffusers_load_directory(directory: Path) -> AnyModel:
+            load_class = GenericDiffusersLoader(
+                app_config=self._app_config,
+                logger=self._logger,
+                ram_cache=self._ram_cache,
+                convert_cache=self.convert_cache,
+            ).get_hf_load_class(directory)
+            result: AnyModel = load_class.from_pretrained(model_path, torch_dtype=TorchDevice.choose_torch_dtype())
+            return result
```

Comment:
This function is unused - I think the logic to get the loader should be checking if it's a directory? I'm not sure how to fix this myself bc the `diffusers_load_directory` function has a different type signature than the other loader function options.

Reply:
Fixed. Looks like I never wired up that function! In the process I discovered a long-standing bug that would prevent text encoders from being loaded generically. 

I also added a unit test for loading generic diffusers models. To do this, I added the 10 MB "tiny" taesdxl model, which was the smallest loadable diffusers I could find.

Reply:
I don't think we should be adding models to the repo. Feels arbitrary to do it just for this one scenario and not all of them, plus 10MB increases the repo size by like 8%. Don't wanna get in the habit of doing this.

I think the best way to test model loading is to package up a models dir that has samples of various model types and configurations, then run full loading tests locally on one of our own machines or a custom CI runner.

Reply:
I've commented out the unit test for directory loading.

Reply:
Can you please drop the commits that added the model file so it's not in the repo?

Reply:
I've removed the model files from git history and force-pushed